specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBSqlAcceleratorCreate.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01-preview",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "accountName": "ddb1",
+    "createUpdateServiceResourceParameters": {
+      "properties":{
+      "serviceType": "Sqlx",
```

Comment:
Ideally, this should be a part of parameters, and sqlX in url should also be a variable.